docs/document/website/versioned_docs/version-1.8.0/guides/table-readonly-table-customization.md

```diff
@@ -0,0 +1,507 @@
+---
```

Comment:
Please update the sample code and images after completing the review of the sample code(kuc-guides).